src/custom-surf/pages/ServiceTicketDetail.tsx

```diff
@@ -109,9 +122,13 @@ const ServiceTicketDetail = () => {
     const showModal = () => setIsModalVisible(true);
 
     useInterval(async () => {
-        console.log("Refreshing");
-        const ticket = await customApiClient.cimTicketById(id);
-        setTicket(ticket);
+        try {
+            const ticket = await customApiClient.cimTicketByIdNoErrorDialog(id);
+            console.log("Refreshing");
```

Comment:
Isn't console.log considered bad practice in frontend apps?

Reply:
For production, probably yeah. For development it has its usages imho.

Perhaps we could do something like this during the app initialization?
```js
if (environment == "production") {
  // Disable console log
  console.log = function () { };
}
```
 https://stackoverflow.com/a/53476667/1284043

Reply:
Removed this console.log, perhaps the optional logging is something to keep in mind for the v2 app @ricardovdheijden